monorepo/monorepo-ws/apps/federation-gateway/src/dic.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  Configuration,
+  GetSupergraphManager,
+  Storage,
+  SupergraphManager,
+} from '@monorepo-ws/supergraph-manager';
+import {
+  superGraphFile,
+  superGraphFileName,
+  supergraphLocation,
+} from './configuration/config';
+
+const getConfiguration = (): Configuration => {
+  const configuration: Configuration = {
+    supergraphFile: superGraphFile,
+    supergraphFileName: superGraphFileName,
+    supergraphLocation: supergraphLocation,
```

Comment:
Kdyžtak můžeš používat zkrácenej zápis, pokud má klíč stejnej název jako proměnná. Je to pak o něco přehlednější.

```suggestion
    supergraphFile,
    supergraphLocation,
    supergraphFileName,
```